src/main/java/com/ably/kafka/connect/config/ChannelSinkConnectorConfig.java

```diff
@@ -154,12 +136,12 @@ public class ChannelSinkConnectorConfig extends AbstractConfig {
     public static final String BATCH_EXECUTION_THREAD_POOL_SIZE_DEFAULT = "10";
 
     public static final String BATCH_EXECUTION_MAX_BUFFER_SIZE = "batchExecutionMaxBufferSize";
-    public static final String BATCH_EXECUTION_MAX_BUFFER_SIZE_DEFAULT = "1000";
+    public static final String BATCH_EXECUTION_MAX_BUFFER_SIZE_DEFAULT = "100";
     private static final String BATCH_EXECUTION_MAX_BUFFER_SIZE_DOC = "Size of the buffer, records " +
         "are buffered or chunked before calling the Ably Batch REST API";
 
     public static final String BATCH_EXECUTION_MAX_BUFFER_DELAY_MS = "batchExecutionMaxBufferSizeMs";
-    public static final String BATCH_EXECUTION_MAX_BUFFER_DELAY_MS_DEFAULT = "5000";
+    public static final String BATCH_EXECUTION_MAX_BUFFER_DELAY_MS_DEFAULT = "100";
```

Comment:
This is 100, but in https://github.com/ably/kafka-connect-ably/pull/142/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R224 (which I believe this is the default value for) it's still 5000.

Reply:
Done in https://github.com/ably/kafka-connect-ably/pull/142/commits/83672a0b3ba3ea277f88d938d50ac9743e2021e7